src/main/java/de/griefed/serverpackcreator/CLISetup.java

```diff
@@ -0,0 +1,235 @@
+                    minecraftVersion,
+                    modLoader,
+                    modLoaderVersion,
+                    includeServerIcon,
+                    includeServerProperties,
+                    includeStartScripts,
+                    includeZipCreation);
+            appLogger.info(LocalizationManager.getLocalizedString("clisetup.log.info.config.enter"));
+            System.out.print(LocalizationManager.getLocalizedString("clisetup.log.info.answer"));
+        } while (!readBoolean());
+        reader.close();
+
+        if (Reference.filesSetup.writeConfigToFile(
+                modpackDir,
+                buildString(Arrays.toString(tmpClientMods)),
+                buildString(Arrays.toString(tmpCopyDirs)),
+                includeServerInstallation,
+                javaPath,
+                minecraftVersion,
+                modLoader,
+                modLoaderVersion,
+                includeServerIcon,
+                includeServerProperties,
+                includeStartScripts,
+                includeZipCreation
+        )) {
+            appLogger.info(LocalizationManager.getLocalizedString("clisetup.log.info.config.written"));
+        }
+    }
+
+    /** A helper method for config setup. Prompts user to enter the values that will be stored in arrays in config.
+    * @return String List. Returns list with user input values that will be stored in config.
+    */
+    private List<String> readStringArray() {
+        Scanner readerArray = new Scanner(System.in);
+        ArrayList<String> result = new ArrayList<>(1);
+        String stringArray;
+        while (true) {
+            stringArray = readerArray.nextLine();
+            if (stringArray.isEmpty()) {
+                return result;
+            } else {
+                result.add(stringArray);
+            }
+        }
+    }
+
+    /** Converts list of strings into concatenated string.
+     * @param args Strings that will be concatenated into one string
+     * @return String. Returns concatenated string that contains all provided values.
+     */
+    String buildString(String... args) {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append(Arrays.toString(args));
+        stringBuilder.delete(0, 2).reverse().delete(0,2).reverse();
+        return stringBuilder.toString();
+    }
+
+    /** A helper method for config setup. Prompts user to enter boolean values that will be stored in config and checks entered values to prevent storing non-boolean values in boolean variables.
+    * @return Boolean. Converts to boolean and returns value entered by user that will be stored in config.
+    */
+    private boolean readBoolean() {
+        Scanner readerBoolean = new Scanner(System.in);
+        String boolRead;
+        while (true) {
+            boolRead = readerBoolean.nextLine();
+            if (boolRead.matches("1")                                                           ||
```

Comment:
Can I request to add back way to input [Yy]es or [Nn]o in English keyboard layout regardless of the current language chosen as I have designed it initially today in the morning?

Reply:
Sure thing.

Reply:
See https://github.com/Griefed/ServerPackCreator/pull/45/commits/ca2219e48092372cfa7db8f50ccc20d87d2b4e5d